tests/test_udf.py

```diff
@@ -0,0 +1,38 @@
+import numpy as np
+import openeo
+import pytest
+import xarray as xr
+
+from openeo_processes_dask.process_implementations.udf.udf import run_udf
+from tests.general_checks import general_output_checks
+from tests.mockdata import create_fake_rastercube
+
+
+@pytest.mark.parametrize("size", [(6, 5, 4, 4)])
+@pytest.mark.parametrize("dtype", [np.float32])
+def test_run_udf(temporal_interval, bounding_box, random_raster_data):
+    input_cube = create_fake_rastercube(
+        data=random_raster_data,
+        spatial_extent=bounding_box,
+        temporal_extent=temporal_interval,
+        bands=["B02", "B03", "B04", "B08"],
+        backend="dask",
+    )
+
+    udf = """
+from openeo.udf import XarrayDataCube
+def apply_datacube(cube: XarrayDataCube, context: dict) -> XarrayDataCube:
+    return cube
```

Comment:
Isn't this UDF too simple to actually be able verify that it did something?
E.g. I would at least the values with 2 or 10 or -1 so you can actually verify the UDF ran.